src/main/java/se/sundsvall/supportmanagement/integration/db/NotificationRepository.java

```diff
@@ -1,24 +1,48 @@
 package se.sundsvall.supportmanagement.integration.db;
 
-import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
-import org.springframework.data.jpa.repository.JpaRepository;
-import se.sundsvall.supportmanagement.integration.db.model.ErrandEntity;
-import se.sundsvall.supportmanagement.integration.db.model.NotificationEntity;
-
 import java.time.OffsetDateTime;
 import java.util.List;
 import java.util.Optional;
 
-@CircuitBreaker(name = "notificationRepository")
-public interface NotificationRepository extends JpaRepository<NotificationEntity, String> {
-
-	boolean existsByIdAndNamespaceAndMunicipalityIdAndErrandEntityId(String id, String namespace, String municipalityId, String errandId);
+import org.springframework.data.domain.Sort;
+import org.springframework.data.jpa.repository.JpaRepository;
 
-	Optional<NotificationEntity> findByIdAndNamespaceAndMunicipalityIdAndErrandEntityId(String id, String namespace, String municipalityId, String errandId);
+import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
+import se.sundsvall.supportmanagement.integration.db.model.ErrandEntity;
+import se.sundsvall.supportmanagement.integration.db.model.NotificationEntity;
 
-	List<NotificationEntity> findAllByNamespaceAndMunicipalityIdAndOwnerId(String namespace, String municipalityId, String ownerId);
+@CircuitBreaker(name = "notificationRepository")
+public interface NotificationRepository extends JpaRepository<NotificationEntity, String> {
 
-	Optional<NotificationEntity> findByNamespaceAndMunicipalityIdAndOwnerIdAndAcknowledgedAndErrandEntityIdAndType(final String namespace, final String municipalityId, final String ownerId, final boolean acknowledged, final String errandId,
+	boolean existsByIdAndNamespaceAndMunicipalityIdAndErrandEntityId(
+		final String id,
+		final String namespace,
+		final String municipalityId,
+		final String errandId);
+
+	Optional<NotificationEntity> findByIdAndNamespaceAndMunicipalityIdAndErrandEntityId(
+		final String id,
+		final String namespace,
+		final String municipalityId,
+		final String errandId);
+
+	List<NotificationEntity> findAllByNamespaceAndMunicipalityIdAndErrandEntityId(
+		final String namespace,
+		final String municipalityId,
+		final String errandId,
+		final Sort sort);
+
+	List<NotificationEntity> findAllByNamespaceAndMunicipalityIdAndOwnerId(
+		final String namespace,
+		final String municipalityId,
+		final String ownerId);
+
+	Optional<NotificationEntity> findByNamespaceAndMunicipalityIdAndOwnerIdAndAcknowledgedAndErrandEntityIdAndType(
+		final String namespace,
+		final String municipalityId,
+		final String ownerId,
+		final boolean acknowledged,
+		final String errandId,
 		final String type);
 
 	List<NotificationEntity> findByExpiresBefore(final OffsetDateTime expires);
```

Comment:
Just to be consistent may I suggest updating the  `existsByNamespaceAndMunicipalityIdAndOwnerIdAndErrandEntityAndDescription` to 
follow
the 
pattern 
:)